pandas/core/frame.py

```diff
@@ -8116,9 +8116,9 @@ def _series_round(s, decimals):
     # ----------------------------------------------------------------------
     # Statistical methods, etc.
 
-    def corr(self, method="pearson", min_periods=1) -> "DataFrame":
+    def corr(self, method="pearson", min_periods=1, axis=0) -> "DataFrame":
```

Comment:
Could you add type annotations to these arguments (except `self`)?

Reply:
Sure